### PR TITLE
Ticket/15726

### PR DESCRIPTION
**Pull request by v12mike**
Checklist:

- [* ] Correct branch: master for new features; 3.2.x for fixes
- [* ] Tests pass
- [* ] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [* ] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15726

### CODE REVIEW

File: tests/cache/apcu_driver_test.php [hunk 1]
```diff
@@ -49,10 +49,27 @@ static public function setUpBeforeClass()
...
+	public function test_purge()
+	{
+		/* add a cache entry which does not match our key */
+		$foreign_key = 'test_' . $this->driver->key_prefix . 'test';
+		$this->assertSame(true, apcu_store($foreign_key, 0, 600));
+		$this->assertSame(true, apcu_exists($foreign_key));
+
+		parent::test_purge();
+
+		$this->assertSame(true, apcu_exists($foreign_key));
```
[the remark this answers is not in the document]
Comment (v12mike) [reply tests/cache/apcu_driver_test.php]: The (parent) test already checks for the purging of items with the expected key.

File: phpBB/phpbb/cache/driver/apcu.php [hunk 2]
```diff
@@ -25,7 +25,11 @@ class apcu extends \phpbb\cache\driver\memory
 	*/
 	function purge()
 	{
-		apcu_clear_cache();
+	/*
```
[the remark this answers is not in the document]
Comment (v12mike) [reply phpBB/phpbb/cache/driver/apcu.php]: Do you want me to fix that, or will it be taken care of as part of the merge?

File: tests/cache/apcu_driver_test.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nicofuma) [reply tests/cache/apcu_driver_test.php]: ok

File: phpBB/phpbb/cache/driver/apcu.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (Nicofuma) [reply phpBB/phpbb/cache/driver/apcu.php]: if you can fix it it would be awesome

**Comment by marc1706** [comment]
Thanks for your contribution. I have adjusted your commits to reflect your correct email address (see https://help.github.com/articles/setting-your-commit-email-address-in-git/ for an explanation on how to apply this to your local git installation).

**Pull request closed without merging**